pages/_document.page.tsx

```diff
@@ -0,0 +1,18 @@
+
+import Document, { Html, Head, Main, NextScript } from 'next/document';
```

Comment:
```suggestion
import Document, { Html, Head, Main, NextScript } from "next/document";
```
Can take some time out to configure prettier such that all single quotes are changed to double quotes on save.